### PR TITLE
Rest

### PR DESCRIPTION
**Pull request by sbasher314**

### CODE REVIEW

File: Database/database.js [hunk 1]
```diff
@@ -24,4 +25,6 @@ const reviewSchema = new mongoose.Schema({
   location: String
 });
 
-module.exports= mongoose.model('Review', reviewSchema);
+reviewSchema.index({bookId: 1, reviewerId: 1}, {unique: true});
```
Comment (djian001) [inline Database/database.js:28]: Interesting, what is this index for?

File: Database/databaseHelpers/seedDB.js [hunk 2]
```diff
@@ -104,7 +108,16 @@ function seedDatabase() {
...
         db.create(reviewObject, function (err, small) {
-          if (err) return handleError(err);
+          attempted++;
+          if (err) {
+            console.error(err);
+          } else {
+            inserted++;
+          }
+          if (attempted >= total) {
+            console.log(`Database Seeded: attempted ${attempted} / ${total} - ${inserted} inserted`);
```
Comment (djian001) [inline Database/databaseHelpers/seedDB.js:118]: Cool way of checking everything got seeded! You might be able to also just keep track of how many errors you get during creation / console log error at i too.

File: README.md [hunk 3]
```diff
@@ -88,6 +88,19 @@ reviewTitle: string
 date: date,
 overalStars: Number}
 
+* Crud API will let you create, read, update, or delete individual reviews based on book-id and reviewer-id:
+
+  ```http://yourEC2Instance:4001/books/:id/reviews/author/:authorid```
```
Comment (djian001) [inline README.md:93]: I like how you're using 1 URL for all CRUD routes!

File: Server/serverHelpers.js [hunk 4]
```diff
@@ -63,16 +66,63 @@ const arrayOfIdsReviewGetter = (req, res, idArray) => {
...
-          res.setStatus(500);
+          console.error(error);
+          res.status(500);
           res.send(error);
         })
     })
     .catch((error) => {
-      res.setStatus(500);
+      console.error(error);
+      res.status(500);
```
Comment (djian001) [inline Server/serverHelpers.js:76]: I think you can actually chain these together like: res.status(500).send(error)

**Review by djian001 — APPROVED** [review]
great job handling everything in your helpers and just calling them in your server file!

File: Server/serverHelpers.js (hunk 4 above)
Comment (sbasher314) [reply Server/serverHelpers.js:76]: That's good to know, I hadn't remembered that

File: Database/database.js (hunk 1 above)
Comment (sbasher314) [reply Database/database.js:28]: To ensure that there is only one author on each review for each book

**Pull request merged**